app/controllers/api/v1/documents_controller.rb

```diff
@@ -31,6 +33,12 @@ def document_attributes
           move: Move.find(params.dig(:move_id))
         )
       end
+
+      protected
+
+      def set_restricted_request_content_type
+        @restricted_request_content_type = 'multipart/form-data'
+      end
```

Comment:
:+1: this approach lets us have different endpoints with custom content types on each :+1: